### DISCUSSION TITLE
hosting

### DISCUSSION THREAD

**Discussion by timjans01** [plan-player-analytics/Plan#2405] [Archive]
Hey, I like this plugin but I have a question,
can I host the panel on an external site, I have my minecraft server pacakge and my web hosting seperate. I can't easily get extra ports to open it. 
if so how can I do this

TIA,
Tim

**Comment by AuroraLS3** [comment]
If you can move the files automatically between your game server and web hosting (like by automating with a cron job) then it is possible.

- Export of html <https://github.com/plan-player-analytics/Plan/wiki/External-WebServer-Use>

**Comment by timjans01** [comment]
Thanks, I will look into it!
